src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -22,28 +24,12 @@
    */
   private final CANSparkMax m_drumMotor;
   private final CANSparkMax m_feedWheelMotor;
-  private int m_repsAboveDrum = 0;
-  private int m_repsAboveFeed = 0;
-  private final PowerDistributionPanel m_pdp;
+  private int m_drumReps = 0;
+  private int m_feedReps = 0;
 
-  public Revolver(PowerDistributionPanel pdp) {
+  public Revolver() {
```

Comment:
This change looks like it will cause the code to not build, since in `RobotContainer` we're still passing in a PDP. Please update `RobotContainer` accordingly.

Also, remember to always test that your code builds before committing or opening a PR.